imageio/plugins/tifffile.py

```diff
@@ -63,7 +63,8 @@ def load_lib():
 
 
 class TiffFormat(Format):
-    """Provides support for a wide range of Tiff images.
+    """Provides support for a wide range of Tiff images using the tifffile 
```

Comment:
The CI is complaining about formatting. I could ask to run `black`, but it looks like you made this change from the GH interface, let's try removing whitespace :)

```suggestion
    """Provides support for a wide range of Tiff images using the tifffile
```